Buddies/Utilities/LocationPersistence.swift

```diff
@@ -24,7 +24,7 @@ class LocationPersistence : NSObject, CLLocationManagerDelegate {
     init(dataAccessor: DataAccessor = DataAccessor.instance) {
         super.init()
         
-        manager.desiredAccuracy = kCLLocationAccuracyThreeKilometers
```

Comment:
It looks like this should be in the previous commit.

Reply:
you should be in the previous commit